torchvision/prototype/models/resnet.py

```diff
@@ -304,36 +305,81 @@ class Wide_ResNet101_2_Weights(WeightsEnum):
     DEFAULT = IMAGENET1K_V2
 
 
+# It may or may not be a good idea to somehow unify that with _VGG_DOC . It's
+# too early to decide yet.
+_RESNET_DOC = """ResNet-{num_layers} from `Deep Residual Learning for Image Recognition <https://arxiv.org/pdf/1512.03385.pdf>`__.
```

Comment:
Please ignore `_RESNET_DOC`, it's not used anywhere and is replaced by `_make_resnet_docstring`. Will remove